Makefile.in

```diff
@@ -216,15 +216,15 @@ install: noicon_install
 	install -m $(FILE_PERM) imgs/telegram22.png $(DESTDIR)$(DATA_ROOT_DIR_PURPLE)/pixmaps/pidgin/protocols/22/telegram.png
 	mkdir -m $(DIR_PERM) -p $(DESTDIR)$(DATA_ROOT_DIR_PURPLE)/pixmaps/pidgin/protocols/48
 	install -m $(FILE_PERM) imgs/telegram48.png $(DESTDIR)$(DATA_ROOT_DIR_PURPLE)/pixmaps/pidgin/protocols/48/telegram.png
-	mkdir -m $(DIR_PERM) -p $(DESTDIR)$(datarootdir)/appdata
-	install -m $(FILE_PERM) AppStream/telegram-purple.metainfo.xml $(DESTDIR)$(datarootdir)/appdata/telegram-purple.metainfo.xml
+	mkdir -m $(DIR_PERM) -p $(DESTDIR)$(datarootdir)/metainfo
+	install -m $(FILE_PERM) AppStream/telegram-purple.metainfo.xml $(DESTDIR)$(datarootdir)/metainfo/telegram-purple.metainfo.xml
 
 .PHONY: uninstall
 uninstall: noicon_uninstall
 	rm -f $(DESTDIR)$(DATA_ROOT_DIR_PURPLE)/pixmaps/pidgin/protocols/16/telegram.png
 	rm -f $(DESTDIR)$(DATA_ROOT_DIR_PURPLE)/pixmaps/pidgin/protocols/22/telegram.png
 	rm -f $(DESTDIR)$(DATA_ROOT_DIR_PURPLE)/pixmaps/pidgin/protocols/48/telegram.png
-	rm -f $(DESTDIR)$(datarootdir)/appdata/telegram-purple.metainfo.xml
```

Comment:
I'd actually like `uninstall` to delete the old file if it still exists, because in any case it's not supposed to be there anymore.

Apart from that: I didn't expect the path layout to change; thanks for the notification! :)

Reply:
done